init.rb

```diff
@@ -48,6 +48,18 @@
 
 current_flavor = Rock.flavors.current_flavor
 
+#The following option is used to determine which xml importer should be used.
+#Currently castxml is working
+
+configuration_option('TYPELIB_CXX_LOADER', 'string',
+    :default => 'castxml',
+    :possible_values => ['gccxml', 'castxml'],
+    :doc => [
+        "Which xml importer should be used?",
+        "Currently only castxml is working when using master flavor"])
```

Comment:
Well. The next release should definitely not support 14.04 IMO.

Can castxml be built on 14.04 ? It does not exist as ubuntu package there, and the llvm version is too old.